docs/examples/State Estimation Example.ipynb

```diff
@@ -504,6 +528,87 @@
     "display(pd.DataFrame(output_data[\"sym_load\"]))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "b2c7c914",
+   "metadata": {},
+   "source": [
+    "### Calculation Method\n",
+    "\n",
+    "There are two calculation methods available and they are defined as the following enumrations:\n",
+    "* **Iterative linear method**: \n",
+    "    * `CalculationMethod.iterative_linear`: All measurements are linearized before running the method.\n",
+    "* **Newton Raphson method**: The Newton-Raphson method considers the problem as a system of real, non-linear equations. It does not make any assumptions on linearization. It could be more accurate and converge in less iterations than the iterative linear method.\n",
+    "    * `CalculationMethod.newton_raphson`: traditional Newton-Raphson method."
```

Comment:
this is repetition of what you said before. please remove and refer to the earlier statements, or make the explanation such that there's no duplication anymore